charts/consul/values.yaml

```diff
@@ -263,6 +275,44 @@ global:
         additionalConfig: |
           {}
 
+      controller:
+        # The directory that Kubernetes will use on Kubernetes CRD creation, 
+        # deletion, and update, to get TLS certificates used issued from vault
+        # to send webhooks to the controller.
+        tlsCert:
+          # The Vault secret path that issues TLS certificates for controller
+          # webhooks.
+          # @type: string
+          secretName: null
+          
+        # The directory that Kubernetes will use on Kubernetes CRD creation, 
+        # deletion, and update, to get CA certificates used issued from vault
+        # to send webhooks to the controller.
+        caCert:
+          # The Vault secret path that contains the CA certificate for controller
+          # webhooks.
+          # @type: string
+          secretName: null
+      
+      connectInject:
+        # The directory that Kubernetes will use on Kubernetes CRD creation, 
```

Comment:
why is it the derectory? Isn't it just pointing to the secret?

Reply:
Let me see.  I'm pretty sure I copied an existing description.  I can fix this.

Reply:
changed.  🙏